docs/source/branch.rst

```diff
@@ -46,6 +46,11 @@ This key refers to an integer that represents the number of different input draw
 
     input_draw_count: 10
 
+.. note::
+    Instead of, or in addition to, specifying an ``input_draw_count``, a list of draws can be specified using the
+    ``input_draws`` key. If ``input_draw_count`` is also specified, the two values are expected to agree, i.e., the
```

Comment:
Suggestion: ``the two values are expected to agree`` -> ``the two values must agree``